src/layouts/guides.html

```diff
@@ -71,6 +76,18 @@
         }
       }
 
+      .content-outlet > div {
+        padding: var(--size-3) 0;
+        position: sticky;
+        bottom: 0;
+        text-align: right;
+        margin: var(--size-px-6) 0 var(--size-px-2) 0;
+      }
+
+      .content-outlet div[class*="edit-on-github"] a {
```

Comment:
hmm, looks like we're piercing into the CE and styling the `<a>` from the outside?  Any reason this style can't be co-located within the _module.css_ file as well?

Reply:
Cleaned this up, and moved to the module file.